doc/code_style/mom.cpp

```diff
@@ -109,10 +106,7 @@ class HalfLife3 : public ValveGame {
 
 			// You may also start one line later
 			// and then indent with a tab:
-			team.relocate(
-				game_engine->get_team()->get_current_hq_location(),
-				gaben.get_table_location()
-			);
+			team.relocate(game_engine->get_team()->get_current_hq_location(), gaben.get_table_location());
```

Comment:
(Just feedback) When something is written in multiple lines usually there is a good reason